message.go

```diff
@@ -523,24 +531,51 @@ func (r *prettyRedisResult) MarshalJSON() ([]byte, error) {
 
 // RedisMessage is a redis response message, it may be a nil response
 type RedisMessage struct {
-	attrs   *RedisMessage
-	string  string
-	values  []RedisMessage
+	attrs *RedisMessage
+	bytes *byte
+	array *RedisMessage
+
+	// integer is used for a simple number or
+	// in conjunction with array or bytes to store the length of array or string
 	integer int64
```

Comment:
```suggestion
	intlen int64
```
Can we change the name of the integer field as well? I am afraid there are some accesses to the integer field that should also be changed.